lib/network/modules/components/shared/Label.js

```diff
@@ -584,7 +645,14 @@ class Label {
         }
       }
     }
-    ctx.font = (values.mod + " " + values.size + "px " + values.face).replace(/"/g, "");
+
+    var fontString = "";
+    if (values.mod !== undefined && values.mod !== "") {  // safeguard for undefined - this happened
+      fontString += values.mod = " ";
```

Comment:
I think you mean `fontString += values.mod + " "` ? It seems really weird to me if you meant to assign `values.mod = " "` and simultaneously add a space to `fontString`

Reply:
Indeed. Good catch.